ethcontract/src/log.rs

```diff
@@ -158,7 +158,12 @@ impl<T: Transport> LogFilterBuilder<T> {
             filter = filter.address(self.address);
         }
         if self.topics != TopicFilter::default() {
-            filter = filter.topic_filter(self.topics)
+            filter = filter.topics(
```

Comment:
Needed because the type would come from a different version of ethabi. This is just a copy of web3's implementation of `topic_filter`.